zerver/migrations/0526_alter_useractivity_id_alter_useractivityinterval_id.py

```diff
@@ -0,0 +1,20 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0525_default_id_autofield"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="useractivityinterval",
+            name="id",
+            field=models.BigAutoField(primary_key=True, serialize=False),
+        ),
+        migrations.AlterField(
+            model_name="useractivity",
+            name="id",
+            field=models.BigAutoField(primary_key=True, serialize=False),
+        ),
```

Comment:
Could consider putting `atomic=False`, so that a failure to migrate the second makes the first complete. But I think we can just stop those two queue workers and it should finish quickly in any case.